packages/catalog-realm/catalog-app/listing/listing.gts

```diff
@@ -70,13 +70,37 @@ class EmbeddedTemplate extends Component<typeof Listing> {
   });
 
   _create = task(async (realmUrl: string) => {
```

Comment:
can u fill all the required specs in the blogApp example so we can prove that this works